applications/MappingApplication/custom_mappers/coupling_geometry_mapper.cpp

```diff
@@ -90,7 +113,7 @@ void CouplingGeometryLocalSystem::CalculateAll(MatrixType& rLocalMappingMatrix,
         rOriginIds[i] = r_geometry_master[i].GetValue(INTERFACE_EQUATION_ID);
     }
     for (IndexType i=0; i< sf_values_slave.size2(); ++i) {
-        rDestinationIds[0] = r_geometry_master[i].GetValue(INTERFACE_EQUATION_ID);
+        rDestinationIds[i] = r_geometry_slave[i].GetValue(INTERFACE_EQUATION_ID);
```

Comment:
@peterjwilson is this correct?
It looks like a copy-paste error but I am not entirely sure